frontend/src/components/PLAYGROUND/ReportSelector/reportSelectorConfig.tsx

```diff
@@ -0,0 +1,184 @@
+import { IReportCategoriesAndTypes } from "types";
+
+export const categoriesAndTypes: IReportCategoriesAndTypes = [
+  {
+    value: "individual_project_reports",
+    label: "Individual Project Reports",
```

Comment:
i like this organization

'types' is a bit confusing, as a property name, but I get the idea

Reply:
agreed